src/main/java/org/broadinstitute/hellbender/tools/exome/ReCapSegCaller.java

```diff
@@ -65,7 +68,7 @@ private ReCapSegCaller() {} // prevent instantiation
 
         // Get the standard deviation of targets belonging to high-confidence neutral segments
         final double[] nearNeutralCoverage = segments.stream()
-                .filter(s -> SegmentUtils.meanTargetCoverage(s, targets)/neutralCoverage < COPY_NEUTRAL_CUTOFF && SegmentUtils.meanTargetCoverage(s, targets)/neutralCoverage > 1/COPY_NEUTRAL_CUTOFF)
+                .filter(s -> (SegmentUtils.meanTargetCoverage(s, targets) - neutralCoverage) < COPY_NEUTRAL_CUTOFF_HIGH && (SegmentUtils.meanTargetCoverage(s, targets) - neutralCoverage) > COPY_NEUTRAL_CUTOFF_LOW)
```

Comment:
Once you have a single cutoff parameter as above, the filter can be written as a single predicate using an absolute value.